doc/01-user-guide.adoc

```diff
@@ -499,17 +499,26 @@ Use `exists?` to check for element existence:
 :xpath-sel: https://www.w3schools.com/xml/xpath_syntax.asp
 :css-sel: https://www.w3schools.com/cssref/css_selectors.asp
 
-* `:active` finds the current active element.
-The Google page, for example, automatically places the focus on the search input.
-So there is no need to click on it first:
+* `:active` finds the currently active element.
+Note that querying for `:active` is deprecated.
+Users are encouraged to call `get-active-element` to retrieve the active element.
+The Google homepage, for example, automatically places the focus on the search input, so there is no need to click on it first.
 +
 [source,clojure]
 ----
+;; Deprecated
 (e/go driver "https://google.com")
 (e/fill driver :active "Let's search for something" k/enter)
+;; Better
+(e/go driver "https://google.com")
+(e/fill-el driver (e/get-active-element driver) "Let's search for something" k/enter)
+;; This case is so common that the API includes `fill-active`
+(e/go driver "https://google.com")
+(e/fill-active driver "Let's search for something" k/enter)
 ----
 
-* any other keyword is translated to an html id attribute:
+* any other keyword is translated to an HTML ID attribute.
+Note that this means that you cannot query for an HTML element with an ID of "active".
```

Comment:
This might imply you cannot search for an id of "active".
Maybe rephrase to something like?:
This means you cannot search for an id of `:active` and would instead have to resort to a longer form like `{:id :active}`.